e2e/src/main/java/gov/hhs/cdc/trustedintermediary/e2e/HttpClient.java

```diff
@@ -13,7 +13,7 @@
 import org.apache.hc.core5.http.io.entity.StringEntity;
 
 /** Mocks a client sending a request to the API * */
-public class Client {
+public class HttpClient {
 
     private static final String DEFAULT_PROTOCOL_DOMAIN = "http://localhost:8080";
     private static String protocolDomain = DEFAULT_PROTOCOL_DOMAIN;
```

Comment:
I know this doesn't include any of the changes, but is there a reason to use `protocolDomain` instead of `DEFAULT_PROTOCOL_DOMAIN` ?

Reply:
I wrote that a while ago.  I did that because of the off chance that we can change `protocolDomain` to something different than the default.  There isn't a way to do that right now with how the code is written, but a setter could be created that overwrites `protocolDomain`.